roles/filesystem/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+filesystem_address_scope: filesystem-address-scope
+filesystem_subnet_pool: filesystem-subnet-pool
+filesystem_ganesha_network: filesystem-ganesha
+filesystem_ganesha_router: filesystem-ganesha-router
+enable_manila_nfs_ganesha_service: "{{ enable_manila_backend_cephfs_nfs | bool }}"
```

Comment:
is a default set here? I'm not sure what the default is in kolla-ansible

Reply:
kolla-ansible has default as false. https://github.com/ChameleonCloud/kolla-ansible/blob/chameleoncloud/xena/ansible/group_vars/all.yml#L692